plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/action/DeleteQueryGroupResponse.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm.action;
+
+import org.opensearch.cluster.metadata.QueryGroup;
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.rest.RestStatus;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+/**
+ * Response for delete QueryGroup
+ *
+ * @opensearch.experimental
+ */
+public class DeleteQueryGroupResponse extends ActionResponse implements ToXContent, ToXContentObject {
+    private final QueryGroup queryGroup;
+    private final RestStatus restStatus;
```

Comment:
I am wondering if we should be using `AcknowledgedResponse` similar to `DeleteIndex` API?

Reply:
I think it should be find to keep ActionResponse. I see `DeletePitResponse` class is also using ActionResponse

Reply:
I think the response of `DeletePitResponse` is slightly different. It has `List<DeletePitInfo>` containing the success boolean and pit id. Doesn't provide the description for each of the Pit. @kaushalmahi12 / @jed326 - What do you guys think?

Reply:
@jainankitk that's a good callout. I think since you're ultimately submitting cluster state update tasks in this API (as well as the create API), then you should be using the `AcknowledgedResponse` and also implement the corresponding `master_timeout`, etc. parameters so that the request does not hang forever on the master response.

Reply:
So should I also change the create response to `AcknowledgedResponse`?

Reply:
@ruai0511 - Yeah, that would make more sense IMO! Probably you can use `DeleteIndex` API as reference?

Reply:
Sorry, I misread that you're asking about create response. That can stay as is, generally it is okay to return the object created, but not required in case of delete!

Reply:
Sorry for chiming in late, let's limit the scope of this PR to the delete API but let's also make sure all of the APIs have a consistent experience with respect to all of the query parameters (timeout, etc.)